server/portal/settings/settings_default.py

```diff
@@ -28,6 +28,7 @@
 
 _RT_QUEUE = 'Web & Mobile Apps'
 _RT_TAG = 'core_portal'
+_CSRF_TRUSTED_ORIGINS = ['https://cep.test']
```

Comment:
Can you add this to `settings_custom.example.py` as well?

Reply:
Added